serializers.md

```diff
@@ -178,38 +178,66 @@ This is why ArraySerialzier is not recommended, but if you are not using meta da
 
 ## JsonApiSerializer
 
-This is a work in progress representation of the [JSON-API] standard. It is included as it is partially working, but has some work left.
+This is a representation of the [JSON-API] standard (v1.0). It implements the most common features such as
 
-There are few differences with the `JsonApiSerializer`. The first is that it uses "side-loading" to include
-other related resources, which is different from the "embedding" approach that is used to include resources
-by the other two serializers.
+ - Primary Data
+ - Resource Objects
+ - Resource Identifier Objects
+ - Compound Documents
+ - Meta Information
+ - Links
+ - Relationships
+ - Inclusion of Related Resources
 
-The second is that it requires a _Resource Key_, which the other two do not.
+Features that are not yet included
+
+ - Sparse Fieldsets
+ - Sorting
+ - Pagination
+ - Filtering
+
+As Fractal is a library to output data structures, the serializer can only transform the content of your HTTP response. Therefore, the following has to be implemented by you
+
+ - Content Negotiation
+ - HTTP Response Codes
+ - Error Objects
+
+For more information please refer to the official [JSON API specification](http://jsonapi.org/format).
+
+JSON API requires a _Resource Key_ for your resources, as well as an _id_ on every object.
 
 ~~~ php
 use League\Fractal\Serializer\JsonApiSerializer;
 $manager->setSerializer(new JsonApiSerializer());
 
 // Important, notice the Resource Key in the third parameter:
-$resource = new Item($book, new GenericBookTransformer(), 'book');
-$resource = new Collection($books, new GenericBookTransformer(), 'books');
+$resource = new Item($book, new JsonApiBookTransformer(), 'books');
+$resource = new Collection($books, new JsonApiBookTransformer(), 'books');
 ~~~
 
-That resource key is used to give it a named namespace:
+The resource key is used to give it a named namespace:
 
 ~~~ php
 // Item
 [
-    'book' => [
-        'foo' => 'bar'
+    'data' => [
+        'type' => 'books',
+        'id' => 1,
+        'attributes' => [
+            'foo' => 'bar'
+        ],
     ],
 ];
 
 // Collection
 [
-    'books' => [
+    'data' => [
         [
```

Comment:
It is a JSON object. The fractal docs include samples in PHP array notation form. The corresponding JSON would have `{`.